packages/amplify-category-auth/src/provider-utils/awscloudformation/index.js

```diff
@@ -249,15 +256,34 @@ function formatCredsforEnvParams(currentEnvSpecificValues, result, resourceParam
 function parseCredsForHeadless(mergedValues, envParams) {
   const oAuthProviders = JSON.parse(mergedValues.hostedUIProviderMeta).map(h => h.ProviderName);
   envParams.hostedUIProviderCreds = JSON.stringify(
-    oAuthProviders.map(el => ({
-      ProviderName: el,
-      client_id: mergedValues[`${el.toLowerCase()}AppIdUserPool`],
-      client_secret: mergedValues[`${el.toLowerCase()}AppSecretUserPool`],
-    })),
+    oAuthProviders.map(el => {
+      if (el === 'SignInWithApple') {
+        return {
+          ProviderName: el,
+          client_id: mergedValues[`${el.toLowerCase()}ClientIdUserPool`],
```

Comment:
Same comment here about all of the repeated `el.toLowerCase()` calls.

Reply:
updated